src/test/java/project/service/GasStationServiceImplTest.java

```diff
@@ -18,76 +18,104 @@
 import java.math.RoundingMode;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.NoSuchElementException;
 
-import static org.mockito.Mockito.when;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
 
 @ExtendWith(MockitoExtension.class)
 class GasStationServiceImplTest {
-
+    @InjectMocks
```

Comment:
In this case is wrong because i have setup method, when i checkout the file i forgot to correct it. In this class in useless. But in the GasStationControllerTest is useful because it will automatically put inject the mocked class into test class constructor. No need to initialise it manual with setup method.
Its corrected.